test/commands/group/removeadmins.test.ts

```diff
@@ -1,66 +0,0 @@
-import {expect} from "@oclif/test";
```

Comment:
Was removal of this test file intentional?

Reply:
Ya, I'm not sure what happened, but we already have a `removeadmin.test.ts` (edited just above this) so I'm not sure why we had a duplicate test file for the same functionality.